src/settings/SettingsStore.ts

```diff
@@ -654,6 +654,43 @@ class A5eGameSettings extends TJSGameSettings {
           type: String
         }
       },
+      // VRC Settings
+      {
+        namespace,
+        key: 'hideVRCProficiencies',
+        options: {
+          name: 'A5E.settings.hideVRCProficiencies',
+          hint: 'A5E.settings.hints.hideVRCProficiencies',
+          scope: scope.world,
+          config: true,
+          default: true,
+          type: Boolean
+        }
+      },
+      {
+        namespace,
+        key: 'hideVRCSpecialties',
+        options: {
+          name: 'A5E.settings.hideVRCSpecialties',
+          hint: 'A5E.settings.hints.hideVRCSpecialties',
+          scope: scope.world,
+          config: true,
+          default: true,
+          type: Boolean
+        }
+      },
+      {
+        namespace,
+        key: 'hideVRCSkills',
+        options: {
+          name: 'A5E.settings.hideVRCSkills',
+          hint: 'A5E.settings.hints.hideVRCSkills',
+          scope: scope.world,
+          config: true,
+          default: true,
+          type: Boolean
+        }
+      },
```

Comment:
Nit. I think it would be better for these to be named `enableVRC...` based on the fact that we hide them by default.